pkg/controllers/rollout/controller_integration_test.go

```diff
@@ -607,6 +607,114 @@ var _ = Describe("Test the rollout Controller", func() {
 		}, 5*time.Minute, interval).Should(BeTrue(), "rollout controller should roll all the bindings to use the latest resource snapshot")
 	})
 
+	It("Rollout should be blocked, then unblocked by eviction", func() {
```

Comment:
need to replace this with a bound binding being evicted

Reply:
Added another IT for bound binding, along with E2E